Gemfile

```diff
@@ -40,7 +40,10 @@ gem 'bootsnap', '>= 1.1.0', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
-
+  #rspec
+  gem 'rspec-rails', '~> 3.8'
+  gem 'shoulda-matchers', '~> 3.1', '>= 3.1.2'
+  gem 'factory_bot_rails', '~> 4.11'
```

Comment:
this gems you won't use in the `development` environment. You can put them in the `test` block with `faker` and `database_cleaner`